backends/dpdk/dpdkArch.cpp

```diff
@@ -984,6 +994,16 @@ CollectLocalVariables::postorder(IR::PathExpression *p) {
     return p;
 }
 
+const IR::Node *
+CollectLocalVariables::postorder(IR::Member *m) {
+    auto expr = m->expr->to<IR::Member>();
+    if (expr && expr->expr->toString() == "m")
```

Comment:
A single line comment saying this is for flattening nested struct would be helpful here